### PR TITLE
Add instruction for creating a none-k8s VM using CAPO

### PR DESCRIPTION
**Pull request by Xenwar**

### CODE REVIEW

File: capo/provision_none_k8s_node.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  labels:
+    cluster.x-k8s.io/cluster-name: basic-1
+  name: clouddata-secret-1
+  namespace: default
+data:
+  value: <Encoded cloud init data>
+```
+
+Then we create the ```mMchine``` and ```Openstackmachine``` resources. Note that 
+the ```KubeadmConfig``` in the ```Machine``` resource is already created during provsioning of the cluster.
```
Comment (furkatgofurov7) [inline capo/provision_none_k8s_node.md:42]:
```suggestion
the ```KubeadmConfig``` in the ```Machine``` resource is already created during provisioning of the cluster.
```

File: capo/provision_none_k8s_node.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: basic-1
+  name: clouddata-secret-1
+  namespace: default
+data:
+  value: <Encoded cloud init data>
+```
+
+Then we create the ```mMchine``` and ```Openstackmachine``` resources. Note that
```
Comment (kashifest) [inline capo/provision_none_k8s_node.md:41]: Machine

File: capo/provision_none_k8s_node.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  name: clouddata-secret-1
+  namespace: default
+data:
+  value: <Encoded cloud init data>
+```
+
+Then we create the ```mMchine``` and ```Openstackmachine``` resources. Note that 
+the ```KubeadmConfig``` in the ```Machine``` resource is already created during provsioning of the cluster.
+
+Machine
```
Comment (furkatgofurov7) [inline capo/provision_none_k8s_node.md:44]: could be more descriptive

File: capo/provision_none_k8s_node.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  cloudsSecret:
+    name: basic-1-cloud-config
+    namespace: default
+  flavor: 2C-4GB-100GB
+  image: Ubuntu_20.04_node
+  sshKeyName: xenwar-key
+```
+
+## Result
+As descussed above, it is possible to create a VM by using CAPO utilities.
```
Comment (furkatgofurov7) [inline capo/provision_none_k8s_node.md:89]:
```suggestion
As discussed above, it is possible to create a VM by using CAPO utilities.
```

Comment (kashifest) [inline capo/provision_none_k8s_node.md:89]: discussed

File: capo/provision_none_k8s_node.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+The created machine has the following properies:
```
Comment (furkatgofurov7) [inline capo/provision_none_k8s_node.md:90]:
```suggestion
The created machine has the following properties:
```

**Review by kashifest — CHANGES_REQUESTED** [review]
A few typos. Otherwise lgtm

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]
Good job!
Small nits

File: capo/provision_none_k8s_node.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  name: osm-director
+  namespace: default
+spec:
+  cloudName: mycloud_default
+  cloudsSecret:
+    name: basic-1-cloud-config
+    namespace: default
+  flavor: 2C-4GB-100GB
+  image: Ubuntu_20.04_node
+  sshKeyName: xenwar-key
```
Comment (furkatgofurov7) [inline capo/provision_none_k8s_node.md:85]: Is this a public key?

Comment (furkatgofurov7) [reply capo/provision_none_k8s_node.md:85]: Sorry, it is just name, disregard.

File: capo/provision_none_k8s_node.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
+# Investigation for deploying non K8S VMs
+
+In order to create a VM that is not part of kubernetes cluster using the CAPO controllers, the following resources created.
```
Comment (smoshiur1237) [inline capo/provision_none_k8s_node.md:3]:
```suggestion
In order to create a VM that is not part of kubernetes cluster using the CAPO controllers, the following resources are created. 
```

**Review by smoshiur1237 — COMMENTED** [review]
Small nits, otherwise Looks good to me

File: capo/provision_none_k8s_node.md (hunk 6 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md:85]: replaced

File: capo/provision_none_k8s_node.md (hunk 7 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md:3]: fixed

File: capo/provision_none_k8s_node.md (hunk 5 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md]: fixed

File: capo/provision_none_k8s_node.md (hunk 4 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md]: fixed

File: capo/provision_none_k8s_node.md (hunk 3 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md:45]: changed

File: capo/provision_none_k8s_node.md (hunk 1 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md]: changed

File: capo/provision_none_k8s_node.md (hunk 4 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md]: fixed

File: capo/provision_none_k8s_node.md (hunk 2 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md]: fixed

File: capo/provision_none_k8s_node.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: basic-1
+  name: clouddata-secret-1
+  namespace: default
+data:
+  value: <Encoded cloud init data>
+```
+
+Then we create the ```Machine``` and ```Openstackmachine``` resources.
```
Comment (fmuyassarov) [inline capo/provision_none_k8s_node.md:41]:
```suggestion
Then we create the ```Machine``` and ```OpenstackMachine``` resources.
```

**Review by fmuyassarov — APPROVED** [review]
just a single nit, otherwise LGTM

File: capo/provision_none_k8s_node.md (hunk 8 above)
Comment (Xenwar) [reply capo/provision_none_k8s_node.md]: fixed

**Review by smoshiur1237 — APPROVED** [review]

**Review by namnx228 — APPROVED** [review]
LGTM

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Review by fmuyassarov — COMMENTED** [review]
LGTM

**Review by furkatgofurov7 — APPROVED** [review]

**Review by kashifest — APPROVED** [review]

**Pull request merged**